docker-entrypoint.sh

```diff
@@ -8,6 +8,9 @@ else
     echo "Skipping cron setup"
 fi
 
+# Initialize app with initial diff check on watched pages
+echo "Initial check on watched pages" 
+bundle exec rake check:all
```

Comment:
question! would this be better suited for the `if [[ "$USE_CRON" == "true" ]]; then` block? That way, once the ECS task scheduler is set up and we remove the line setting `USE_CRON` to `true`, we won't have to relocate this? Because I imagine similarly to how it's slightly preferable to only have local `rake check:all` calls made in the cronjob (or close to it), it'd also be preferable to only run the page checks with the ecs scheduler in the deployed env.

Reply:
Ahh, you're totally right. When I initially put this together, I was thinking that we're run this initial `rake check:all` in both environments. But you remind me that we only really need to do this in the local env.